src/components/validation/runner/mod.rs

```diff
@@ -18,7 +18,7 @@ use super::{
     ComponentType, TestCaseExpectation, TestEvent, ValidationConfiguration, Validator,
 };
 
-use self::config::TopologyBuilder;
+pub use self::config::*;
```

Comment:
I would strongly discourage importing `*`. It could lead to problems down the line when someone adds a new import to `config` that clashed with something else imported here. Also reduces greppability.

Reply:
Fixed, thanks!